### DISCUSSION TITLE
Nextjs shows weird error

### DISCUSSION THREAD

**Discussion by devmnj** [mantinedev/mantine#3543] [Q&A]
When I start using with Mantine with Nextjs I goit this err

`./src/components/ColorSchemeToggle.tsx:3:2
Module not found: Package path . is not exported from package D:\manine2\node_modules\@tabler\icons (see exports field in D:\manine2\node_modules\@tabler\icons\package.json)
  1 | import { ActionIcon, Group, useMantineColorScheme } from '@mantine/core';
  2 | import dynamic from 'next/dynamic';
> 3 |   import { IconSun, IconMoonStars } from '@tabler/icons';
    |  ^
  4 |  
  5 | export function ColorSchemeToggle() {
  6 |   const { colorScheme, toggleColorScheme } = useMantineColorScheme();

Import trace for requested module:
./src/pages/index.tsx

https://nextjs.org/docs/messages/module-not-found`

Could anyone fix it ?

**Comment by rtivital** [comment]
Use `@tabler/icons-react` package instead
